control-plane/subcommand/server-acl-init/anonymous_token.go

```diff
@@ -7,9 +7,22 @@ import (
 	"github.com/hashicorp/consul/api"
 )
 
+const (
+	AnonymousTokenPolicyName = "anonymous-token-policy"
```

Comment:
Both of these can be a private constants

Reply:
done